tests/src/performance/Scenario/JitBench/Benchmarks/WebAppBenchmarks.cs

```diff
@@ -241,8 +252,8 @@ string GetLocalJitBenchRepoDirectory()
             return Environment.GetEnvironmentVariable("MUSICSTORE_PRIVATE_REPO");
         }
 
-        private const string JitBenchRepoUrl = "https://github.com/aspnet/JitBench";
-        private const string JitBenchCommitSha1Id = "6e1327b633e2d7d45f4c13f498fc27698ea5735a";
+        private const string JitBenchRepoUrl = "https://github.com/adamsitnik/JitBench";
```

Comment:
Once the `aspnet/JitBench` PR is merged, we should change this to use the official repo.

Reply:
You could always do something similar to my "MUSICSTORE_PRIVATE_REPO" env var if you need it for local testing.

Reply:
@noahfalk thanks for the hint!